package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
would you mind switching to eslint? that way linting can be deferred to Lab with the `-L` flag